src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,21 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+
     public static void main(String[] args) {
+        Movie movie = new Movie();
+        movie.setTitle("Terminator-2");
+        movie.setDescription("fantasy film");
+        movieService.add(movie);
+        Movie movie1 = movieService.get(1L);
+        System.out.println(movie1.toString());
```

Comment:
```suggestion
        System.out.println(movie1);
```

toString will be called under the hood